server-toss/.env.example

```
@@ -10,5 +10,6 @@ TOSS_API_BASE_URL=https://apps-in-toss-api.toss.im
 TOSS_API_KEY=TossApiKey
 TOSS_DECRYPT_KEY=TossDecryptKey
 TOSS_DECRYPT_AAD=TOSS
-TOSS_CLIENT_CERT_PATH=./certs/davinci_public.crt
-TOSS_CLIENT_KEY_PATH=./certs/davinci_private.key
+TOSS_CLIENT_CERT_PATH=./certs/davinci_new_public.crt
+TOSS_CLIENT_KEY_PATH=./certs/davinci_new_private.key
+JWT_SECRET=32자 이상 문자열
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_SECRET key should go before the MYSQL_DATABASE key

(UnorderedKey)